server/controllers/finance/reports/collection_capacity/report.handlebars

```diff
@@ -0,0 +1,53 @@
+{{> head title="REPORT.COLLECTION_CAPACITY.TITLE"}}
+
+<body>
+
+<div class="container">
+  {{> header}}
+
+  <!-- body  -->
+  <div class="row">
+    <div class="col-xs-12">
+
+      <!-- page title  -->
+      <h3 class="text-center text-uppercase">
+        <strong>{{translate 'REPORT.COLLECTION_CAPACITY.TITLE'}}</strong> <br>
+        <small>{{date dateFrom}} - {{date dateTo}}</small>
+      </h3>
+
+      <table class="table table-striped table-condensed table-report table-bordered">
```

Comment:
Could you put an explanation at the top of this report to explain the colors?  We should mention the ranges for green, yellow, and red.